src/cpp/session/resources/grid/datatables/js/jquery.dataTables.js

```diff
@@ -1636,7 +1636,7 @@
     if ( row.nTr === null )
     {
       nTr = nTrIn || document.createElement('tr');
-      nTr.setAttribute("data-row", iRow);
+      nTr.setAttribute("data-row", iRow + oSettings._iDisplayStart);
```

Comment:
Found that if you made the window small enough such that it was AJAXing data in constantly the row number would be off here as `iRow` doesn't encapsulate the data pagination.